katib-ui-rock/rockcraft.yaml

```diff
@@ -1,11 +1,11 @@
-# Based on https://github.com/kubeflow/katib/blob/v0.15.0/cmd/ui/v1beta1
+# Based on https://github.com/kubeflow/katib/blob/v0.16.0/cmd/ui/v1beta1/Dockerfile
 name: katib-ui
 summary: Katib UI
 description: |
   Katib UI.
-version: v0.15.0_22.04_1
+version: v0.16.0
```

Comment:
Yeah if canonical/bundle-kubeflow#747 closes quick then we should edit this version, but otherwise maybe we merge this PR as is and edit this again if needed when canonical/bundle-kubeflow#747 is finalised?  

I'm open either way, but to me its better to update this rock now with a best guess at version, then modify the version later as needed